.gitignore

```diff
@@ -41,4 +41,7 @@ npm-debug.log*
 
 # cypress
 cypress/videos
-cypress/screenshots
\ No newline at end of file
+cypress/screenshots
+
+# developer environment
+.vscode/
```

Comment:
I'd just delete it; it should not concern the project, which editor is used by whom; it can be added to "global" .gitignore by each developer

Reply:
Agreed